### ISSUE TITLE
[bug] outline and dynamic animations

### ISSUE THREAD

**Issue by foxyseta** [andreasKroepelin/polylux#146]
# Minimal example

```typst
#import "@preview/polylux:0.3.1": *
#import themes.university: *

#slide(title: "Contents")[
  #polylux-outline()
]

#slide(title: [The open question: $P eq.not NP -> SampNP subset.eq.not AvgP$ ?],
  new-section: Title)[
    #only(1)[a]
    #uncover((beginning: 2))[b]
    #uncover(3)[c]
]
```

# Actual result

The outline features one entry per subslide (so three in total).

# Expected result

The outline features a unique entry for the logical slide.

**Comment by andreasKroepelin** [comment]
Good catch. For further reference (and future me fixing this), here is the culprit:
https://github.com/andreasKroepelin/polylux/blob/main/themes%2Funiversity.typ#L127-L129

**Comment by foxyseta** [comment]
Glad I could help. Love your work!

**Comment by foxyseta** [comment]
It appears it also gives problems with theorems/definitions numbering packages, e.g. https://github.com/typst/packages/tree/main/packages/preview/ctheorems/1.1.0.

**Comment by andreasKroepelin** [comment]
Yeah that's a thing we can't do anything about it here, unfortunately. The Typst team is aware of that, there might be a solution some day :)

**Comment by foxyseta** [comment]
Is there an open issue on the officual typst repository then? Should this be listed under "Known issues" or something?

**Comment by andreasKroepelin** [comment]
See https://github.com/typst/typst/issues/1841

**Comment by foxyseta** [comment]
Thanks! What about content too big for one slide that hence spawns on multiple slides, though? This wouldn't be a state freeze limitation, right?

**Comment by andreasKroepelin** [comment]
No, but that's intentionally allowed. Think long bibliographies, for example.

**Comment by foxyseta** [comment]
Yeah, that's my use case. But that brings about multiple "References" entries in the ToC.

**Comment by andreasKroepelin** [comment]
Oh I see... Yeah that should be fixable.
